translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.translate;
+
+// [START translate_hybrid_imports]
```

Comment:
Can we use the new format and move each snippet into it's own file so we don't need to do this?

Reply:
This one is a tutorial not a snippet, so I wasn't really sure what rules we have for those.

Reply:
Can you link to the tutorial? These feel pretty snippet-y to me - they don't really interact much from what I saw.

Reply:
I believe it is this one: https://cloud.google.com/translate/docs/hybrid-glossaries-tutorial